.travis.yml

```diff
@@ -17,7 +17,7 @@ before_install:
 android:
   components:
     - platform-tools
-    - build-tools-28.0.3
+    - build-tools-29.0.2
```

Comment:
We tried this before, but not available on Travis yet :\

Reply:
looks like it is available 🤷‍♂ 
![image](https://user-images.githubusercontent.com/1120468/67386333-0c7d9600-f595-11e9-9278-f6d22c708e34.png)

Reply:
@bruno-garcia yeah I told @tonyo about that, but we wanted to try it again as it shows in the log

Reply:
I see. So it was finally added?!
Builds are hanging though

Reply:
build passed and build logs also show them being downloaded